lib/api/apiUtils/authorization/permissionChecks.js

```diff
@@ -2,19 +2,32 @@ const { evaluators, actionMaps, RequestContext } = require('arsenal').policies;
 const constants = require('../../../../constants');
 
 const { allAuthedUsersId, bucketOwnerActions, logId, publicId,
-    assumedRoleArnResourceType, backbeatLifecycleSessionName } = constants;
+    assumedRoleArnResourceType, backbeatLifecycleSessionName, arrayOfAllowed } = constants;
 
 // whitelist buckets to allow public read on objects
 const publicReadBuckets = process.env.ALLOW_PUBLIC_READ_BUCKETS ?
     process.env.ALLOW_PUBLIC_READ_BUCKETS.split(',') : [];
 
-function checkBucketAcls(bucket, requestType, canonicalID) {
+function checkBucketAcls(bucket, requestType, canonicalID, mainApiCall) {
```

Comment:
I am not sure where the `mainApiCall` comes from and where it is set. JSDOC might be helpful.

Reply:
JSDOC has been added , let me know if it's still not clear enough

Reply:
I do not see it defined here: https://github.com/scality/cloudserver/blob/e0eab954aad6ced74c3a0a5d9b0517808dcf8c86/lib/api/apiUtils/authorization/permissionChecks.js#L315

Is it on purpose?

Reply:
As it's not effecting the current behavior , it's be defined on this PR : https://github.com/scality/cloudserver/pull/5432 as we're trying to do small/1 logical change per PR